src/MongoDB/BulkWrite.c

```diff
@@ -168,12 +168,13 @@ static bool php_phongo_bulkwrite_delete_apply_options(bson_t *boptions, zval *zo
    Constructs a new BulkWrite */
 static PHP_METHOD(BulkWrite, __construct)
 {
-	php_phongo_bulkwrite_t  *intern;
-	zend_error_handling      error_handling;
-	zval                    *options = NULL;
-	zend_bool                ordered = 1;
-	SUPPRESS_UNUSED_WARNING(return_value_ptr) SUPPRESS_UNUSED_WARNING(return_value) SUPPRESS_UNUSED_WARNING(return_value_used)
-
+	php_phongo_bulkwrite_t* intern;
+	zend_error_handling     error_handling;
+	zval*                   options = NULL;
+	zend_bool               ordered = 1;
+	SUPPRESS_UNUSED_WARNING(return_value_ptr)
```

Comment:
I think these would be best put in their own paragraph.

I'm not sure if it impacts `clang-format` at all, as it probably considers them function calls.